wagtail/contrib/wagtailfrontendcache/signal_handlers.py

```diff
@@ -14,6 +36,20 @@ def page_unpublished_signal_handler(instance, **kwargs):
     purge_page_from_cache(instance)
 
 
+def page_pre_saved_signal_handler(instance, **kwargs):
+    if has_changed(instance, 'url_path'):
+        old_instance = get_old_instance(instance)
+        instance.old_url = old_instance.full_url
+    else:
+        instance.old_url = None
+
+
+def page_post_saved_signal_handler(instance, **kwargs):
```

Comment:
@kaedroho we can't because in the page_published signal the old url has been overwritten already. The only place I could still get it was pre_save signal :(